WORKSPACE

```diff
@@ -67,9 +67,9 @@ http_archive(
 
 http_archive(
     name = "qsim",
-    sha256 = "f390ee72cf88c48d81c98262c599dc45d660a2a9308a9ee903bfa73aec08a9b4",
-    strip_prefix = "qsim-0.6.0",
-    urls = ["https://github.com/quantumlib/qsim/archive/v0.6.0.zip"],
+    sha256 = "e2853379bde52d6277f9be4b80f54d32b3b27f7242a6c561cb34fb12d823b80e",
```

Comment:
There is some weird bleedover from old PRs working it's way into this one. And currently the CI tests aren't passing. I think this PR might've worked it's way into a pretty ugly state. Maybe it's worth starting a fresh branch off master and putting the changes you want in again ?